.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,55 @@
+# Implementing a new supported platform
+
+## At Development
+Create PR's addressing the following:
+- [ ] [coreos-assembler](https://github.com/coreos/coreos-assembler)
+    - [Example PR](https://github.com/coreos/coreos-assembler/pull/2489)
+    - [ ] `Implement required functionality to support new platform`
+- [ ] [fedora-web](https://pagure.io/fedora-web/websites)
```

Comment:
Let's move this way down to the bottom of the list.  It's not the primary thing implementors should be thinking about.

Reply:
Okay!